build.cmd

```diff
@@ -1,3 +1,4 @@
 @ECHO OFF
 SETLOCAL
-PowerShell -NoProfile -NoLogo -ExecutionPolicy unrestricted -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = ''; [System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; try { & '%~dp0build.ps1' %*; exit $LASTEXITCODE } catch { write-host $_; exit 1 }"
+PowerShell -NoProfile -NoLogo -ExecutionPolicy Bypass -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = ''; [System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; try { & '%~dp0build.ps1' %*; exit $LASTEXITCODE } catch { write-host $_; exit 1 }"
```

Comment:
I looked at the build scripts of CoreFX and everywhere else in arcade and it seems like ByPass is used as the execution policy instead of unrestricted. I'm not sure if this will help but it should be safe to try.

Reply:
FYI ByPass tends to work better in non-interactive cases because `powershell` will never prompt a non-existing user to see if it should continue. But, I doubt this change will help _complete_ scripts more reliably.